_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -5917,25 +5917,27 @@
 /turf/open/floor/iron/dark,
 /area/station/security/brig)
 "avH" = (
-/obj/structure/curtain,
 /obj/item/soap/deluxe,
 /obj/item/bikehorn/rubberducky,
-/obj/machinery/shower/directional/north,
+/obj/machinery/shower/directional/east,
+/obj/machinery/door/window/left/directional/east{
+	req_access = list("captain");
```

Comment:
Use mapping access helpers instead

Reply:
Just tried that and it didn't work. Most other access-bound windoors on Pubby station are like this as well.